src/cascadia/TerminalControl/IControlSettings.idl

```diff
@@ -32,7 +32,8 @@ namespace Microsoft.Terminal.Control
         String ProfileName;
         String ProfileSource;
 
-        Boolean EnableUnfocusedAcrylic;
+        Boolean EnableUnfocusedAcrylic { get; };
+        Guid SessionId { get; };

```

Comment:
I didn't see this one get used; do we use it?

Reply:
Yes, via the `TerminalSettings` object to get the `sessionId` into the connection.